### PR TITLE
#420 Adding canonical refs to ./getting-started part 3/10

### PR DESCRIPTION
**Pull request by martyav**
See https://github.com/rancher/rancher-docs/pull/619 for more info

### CODE REVIEW

File: docs/getting-started/installation-and-upgrade/other-installation-methods/air-gapped-helm-cli-install/docker-install-commands.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 title: Docker Install Commands
 ---
 
+<head>
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/getting-started/installation-and-upgrade/other-installation-methods/air-gapped-helm-cli-install/docker-install-commands"/>
```
Comment (btat) [inline docs/getting-started/installation-and-upgrade/other-installation-methods/air-gapped-helm-cli-install/docker-install-commands.md:6]: For 2.0-2.4, `versioned_docs/version-2.0-2.4/getting-started/installation-and-upgrade/other-installation-methods/air-gapped-helm-cli-install/install-rancher-ha.md` is the corresponding page.

File: docs/getting-started/installation-and-upgrade/installation-requirements/install-docker.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 title: Installing Docker
 ---
 
+<head>
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/getting-started/installation-and-upgrade/installation-requirements/install-dockers"/>
```
Comment (btat) [inline docs/getting-started/installation-and-upgrade/installation-requirements/install-docker.md:6]:
```suggestion
  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/getting-started/installation-and-upgrade/installation-requirements/install-docker"/>
```

File: versioned_docs/version-2.0-2.4/getting-started/installation-and-upgrade/installation-requirements/install-docker.md [hunk 3]
```diff
@@ -2,6 +2,10 @@
 title: Installing Docker
 ---
 
+<head>
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/getting-started/installation-and-upgrade/installation-requirements/install-dockers"/>
```
Comment (btat) [inline versioned_docs/version-2.0-2.4/getting-started/installation-and-upgrade/installation-requirements/install-docker.md:6]:
```suggestion
  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/getting-started/installation-and-upgrade/installation-requirements/install-docker"/>
```

File: versioned_docs/version-2.5/getting-started/installation-and-upgrade/installation-requirements/install-docker.md [hunk 4]
```diff
@@ -2,6 +2,10 @@
 title: Installing Docker
 ---
 
+<head>
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/getting-started/installation-and-upgrade/installation-requirements/install-dockers"/>
```
Comment (btat) [inline versioned_docs/version-2.5/getting-started/installation-and-upgrade/installation-requirements/install-docker.md:6]:
```suggestion
  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/getting-started/installation-and-upgrade/installation-requirements/install-docker"/>
```

File: versioned_docs/version-2.6/getting-started/installation-and-upgrade/installation-requirements/install-docker.md [hunk 5]
```diff
@@ -2,6 +2,10 @@
 title: Installing Docker
 ---
 
+<head>
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/getting-started/installation-and-upgrade/installation-requirements/install-dockers"/>
```
Comment (btat) [inline versioned_docs/version-2.6/getting-started/installation-and-upgrade/installation-requirements/install-docker.md:6]:
```suggestion
  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/getting-started/installation-and-upgrade/installation-requirements/install-docker"/>
```

File: versioned_docs/version-2.0-2.4/installation/resources/chart-options/chart-options.md [hunk 6]
```diff
@@ -2,4 +2,8 @@
 title: Rancher Helm Chart Options
 ---
 
+<head>
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/getting-started/installation-and-upgrade/installation-references/helm-chart-options"/>
```
Comment (btat) [inline versioned_docs/version-2.0-2.4/installation/resources/chart-options/chart-options.md:6]: Need to add to the corresponding 2.5, 2.6, /docs pages.

**Review by btat — CHANGES_REQUESTED** [review]

File: versioned_docs/version-2.0-2.4/installation/resources/chart-options/chart-options.md (hunk 6 above)
Comment (martyav) [reply versioned_docs/version-2.0-2.4/installation/resources/chart-options/chart-options.md:6]: These were actually already addressed in https://github.com/rancher/rancher-docs/pull/604

**Comment by martyav** [comment]
https://github.com/rancher/rancher-docs/pull/604 covered the seemingly-missing helm-chart-options pages, and [1fb63c7](https://github.com/rancher/rancher-docs/pull/622/commits/1fb63c7bb3c5cbc756036530537fee19700e425d) addresses other comments

**Review by btat — APPROVED** [review]

**Pull request merged**